src/core/store/columnContext.tsx

```diff
@@ -14,6 +14,7 @@ export type PanelMeta = {
   closeCb: () => void;
   minHeight: number;
   getOpenState: () => boolean;
+  debug_caller?: string;
```

Comment:
What the reason to use camel_case naming here?

Reply:
And i can get how and for what this used

Reply:
to draw attetntion on debug caller
Then i left this key and by doing that i'm asking - should i keep it or remove it?

Reply:
it helps when you debugging cards/panels - they don't have names, only minHeight and Jsx elements

Reply:
Don't leave it if that's not used, or implement debug tool fully, or remove it.

Reply:
it's very nice suggestion about just `id`! I wonder how i haven't thought about it (probably because this id doesn't utilized anywhere)